pages/named-witches.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div style="height: 100%;">
+    <div id="main-content" class="pl-5 pr-5 pt-3 pb-3">
+      <div class="flex content-start items-center">
+        <h1 class="text-sm md:text-xl lg:text-2xl">Trial Mentions Network Graph</h1>
+        <span class="rounded-full border-r border-l border-gray-400 w-6 h-6 flex items-center justify-center ml-2">
+          <!-- icon by feathericons.com -->
+          <svg aria-hidden="true" class="" data-reactid="266" fill="none" height="24" stroke="#606F7B" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg" @click="showPageInfo()">
+            <line x1="12" y1="16" x2="12" y2="12"></line><line x1="12" y1="8" x2="12" y2="8"></line>
+          </svg>
+        </span>
+      </div>
+
+    </div>
+    <div style="height:90%; min-height: 400px;">
+      <iframe src='https://flo.uri.sh/story/1664928/embed' title='Interactive or visual content' class='flourish-embed-iframe' frameborder='0' scrolling='no' style='width:100%;height:100%;' sandbox='allow-same-origin allow-forms allow-scripts allow-downloads allow-popups allow-popups-to-escape-sandbox allow-top-navigation-by-user-activation'></iframe><div style='width:100%!;margin-top:4px!important;text-align:right!important;'><a class='flourish-credit' href='https://public.flourish.studio/story/1664928/?utm_source=embed&utm_campaign=story/1664928' target='_top' style='text-decoration:none!important'><img alt='Made with Flourish' src='https://public.flourish.studio/resources/made_with_flourish.svg' style='width:105px!important;height:16px!important;border:none!important;margin:0!important;'> </a></div>
```

Comment:
Not sure where the text is coming from. I assume it is within Flourish itself but it says:

>  Press the arrows to see how this changes over time

or something similar. This confused me as there are arrows linking the named witches together so I tried clicking them. I also tried using my keyboard arrows.

I was wondering if we could update the text here to instead say something like.

> Use the navigational arrows to the left to...

Reply:
I don't have access to the flourish but I will see if Ewan does.

Reply:
Got the log in details and have changed the text.

Reply:
Looks good to me. I will approve this but hold off merging until it's parent branch is also ready. Thanks Ruby